frontend/compositions/officer-view/officer-view-header/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { OfficerRecordType } from "../../../models/officer"
+import styles from "./officer-view-header.module.css"
+
+export default function OfficerHeader(officer: OfficerRecordType) {
+  const { firstName, lastName, badgeNo, status, department } = officer
+  const { category, name, titleAndName, otherData } = styles
+
+  return (
+    <div>
+      <h3>Officer Record</h3>
+      <div style={{ display: "flex" }}>
```

Comment:
Should be using css module class even if just one css line. please move to the css module.